apps/api/src/app/change/usecases/create-change/create-change.usecase.ts

```diff
@@ -30,6 +31,10 @@ export class CreateChange {
     if (change) {
       change.change = changePayload;
 
+      if (command.action === ChangeEntityActionEnum.DELETE) {
+        change.action = command.action;
```

Comment:
We are only showing create and delete changes?

Reply:
No we show update as well but if a change will be updated instead and it is a delete we update the action to delete.

Reply:
It is setting it to update here https://github.com/novuhq/novu/pull/4224/files#diff-475c37ecc04947daa8753f461702772f7785813599e76791f8a40c7b16781fb4R133